tsconfig.json

```diff
@@ -10,6 +10,7 @@
   "typedocOptions": {
     "name": "Chart.js",
     "excludeExternals": true,
+    "excludeNotExported": true,
```

Comment:
Does the directly exporting change anything else in the docs? Like not making those "external" or something.

Reply:
No, I think the only change is that it hides some stuff